### ISSUE TITLE
[Bug] The App is not opening (Only dev version)

### ISSUE THREAD

**Issue by admin-elvistrujanovic** [PojavLauncherTeam/PojavLauncher_iOS#286]
### Describe the bug

The application has been constantly crashing since its launch - only two newer dev versions are infected with this bug

### The log file and images/videos

PojavLauncher App: Logs not available Cr4shed App: Cr4shed is installed a long time ago but for this app the bug is not logged by this app

### Steps To Reproduce

```markdown
1. Start PojavLauncher
2. See the PojavLauncher Welcome Screen for 0.5 second
3. The app is closed automatically
```

### Expected Behavior

I expect the app to open

### Platform

```markdown
- Device model: iPhone 7 (GSM)
- iOS version: 14.5.1
- PojavLauncher version: Latest Pre-Release - Dev version
```

### Anything else?

/

**Comment by asdfugil** [comment]
add ` -Xms1M` to the java_args property of the property list /var/mobile/Documents/. pojavlauncher/launcher_preferences.plist and try again

**Comment by admin-elvistrujanovic** [comment]
> add ` -Xms1M` to the java_args property of the property list /var/mobile/Documents/. pojavlauncher/launcher_preferences.plist and try again

Unfortunately, It's not fixed

**Comment by khanhduytran0** [comment]
Try to attach the app and debug the app in LLDB.

**Comment by khanhduytran0** [comment]
*No response moment.*

**Comment by admin-elvistrujanovic** [comment]
> *No response moment.*

Don't worry, I'm fixed this issue with FutureRestore iOS 14.5.1 ipsw with blobs and jailbreak again with Odysseyra1n
